### ISSUE TITLE
Can't install withExEditorHost on Windows 7 Home Premium or Ubuntu 16.04 LTS

### ISSUE THREAD

**Issue by therealdano** [asamuzaK/withExEditorHost#31]
I have two different systems. I am running Windows 7 Home Premium on one and Ubuntu 16.04 LTS on the other. I have tried to install both version 2.2.5 and 2.2.6 of withExEditorHost. When I am instructed to run
`npm run setup`
from the extracted withExEditorHost directory I get the error that the script is missing:
```
npm ERR! missing script: setup

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\dano.touch3\AppData\Roaming\npm-cache\_logs\2018-02-28T22_07_22_385Z-debug.log
```
I have installed Node.js version 8.9.4 LTS. This is a clean install in both cases and no other npm packages are installed.

When I attempt to run
`node setup.js`
I get an error message indicating a missing npm package
```
module.js:540
    throw err;
    ^

Error: Cannot find module 'web-ext-native-msg'
    at Function.Module._resolveFilename (module.js:538:15)
    at Function.Module._load (module.js:468:25)
    at Module.require (module.js:587:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (C:\Users\dano.touch3\withExEditorHost\setup.js:10:7)
    at Module._compile (module.js:643:30)
    at Object.Module._extensions..js (module.js:654:10)
    at Module.load (module.js:556:32)
    at tryModuleLoad (module.js:499:12)
    at Function.Module._load (module.js:491:3)
```
So I install the missing web-ext-native-msg package. I then rerun
`node setup.js`
Which seems to go through the process of setting up the configuration for my Firefox browser. However version 4.2.6 of withExEditor will not accept the install withExEditorHost. I have tried both reloading the addon and restarting Firefox. withExEditorHost says the host version is incompatible. 

The problems are almost identical when trying to install on Ubuntu (paths are different of course).

In summary:
* `npm run setup` fails
* `node setup.js` fails until you run `npm install web-ext-native-msg`
* After a successful setup, withExEditor states withExEditorHost is incompatible.

**Comment by therealdano** [comment]
I found from looking in the package.json from a previously successful install that one of the dependencies is the npm package semver-parser. I installed that, reran `node setup.js` and reloaded withExEditor and it accepted the withExEditorHost install.

I would like to see the installation instructions updated to reflect the manual npm installs needed.

**Comment by asamuzaK** [comment]
I'm sorry.
I'm in the midst of creating version 3, and the README was rewritten halfway.

**Comment by asamuzaK** [comment]
Updated README.

**Comment by therealdano** [comment]
Great! That worked like a charm on the Ubuntu system. Thank you!
